cpp/include/cudf/copying.hpp

```diff
@@ -627,10 +627,25 @@ std::unique_ptr<table> boolean_mask_scatter(
  *
  * @returns Returns a table by scattering `input` into `target` as per `boolean_mask`.
  */
- std::unique_ptr<table> boolean_mask_scatter(
+std::unique_ptr<table> boolean_mask_scatter(
     std::vector<std::reference_wrapper<scalar>> const& input, table_view const& target,
     column_view const& boolean_mask,
     rmm::mr::device_memory_resource *mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Get the element at specified index from a column
+ * 
```

Comment:
We should probably add some documentation warning that this function is expensive (invokes a kernel launch) and should not be (mis)used in performance sensitive code.

Reply:
documentation added as note.